src/fragment-components/tiles/a-clickable-tile.tsx

```diff
@@ -132,15 +132,13 @@ export const componentInfo: ComponentInfo = {
 	codeExport: {
 		angular: {
 			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Href = '${json.href}';
-				@Input() ${nameStringToVariableString(json.codeContext?.name)}Disabled = ${json.disabled || false}`,
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Disabled = ${json.disabled || false}
```

Comment:
Is this missing a semicolon?
```suggestion
				@Input() ${nameStringToVariableString(json.codeContext?.name)}Disabled = ${json.disabled || false};
```